package.json

```
@@ -23,7 +23,7 @@
     "nodeunit-x": "0.15.0"
   },
   "dependencies": {
-    "re2": "1.20.7"
+    "re2": "1.21.4"
```

Also, I was wondering... Why not make this version constraint more relaxed?

```suggestion
    "re2": "^1.21.4"
```

Or is the concern here that newer minor versions of `re2` might break this package?

---

Developers always break their libraries from time to time :) I prefer sticking to a specific version and updating dependencies by myself, so I can run tests and make sure nothing broke in my own project.